controllers/kuadrant_controller.go

```diff
@@ -268,22 +372,76 @@ func (r *KuadrantReconciler) registerExternalAuthorizer(ctx context.Context, kOb
 	return nil
 }
 
-func (r *KuadrantReconciler) reconcileSpec(prevCtx context.Context, kObj *kuadrantv1beta1.Kuadrant) (ctrl.Result, error) {
-	logger, err := logr.FromContext(prevCtx)
-	if err != nil {
-		return ctrl.Result{}, err
+func (r *KuadrantReconciler) registerExternalAuthorizerOSSM(ctx context.Context, kObj *kuadrantv1beta1.Kuadrant) error {
+	logger, _ := logr.FromContext(ctx)
+
+	if err := r.registerToServiceMeshMemberRoll(ctx, kObj); err != nil {
+		return err
+	}
+
+	smcp := &maistrav2.ServiceMeshControlPlane{}
+
+	smcpKey := client.ObjectKey{Name: iopName(), Namespace: iopNamespace()}
```

Comment:
nitpick: `iopName` stands for IstioOperator (object) Name. Maybe we should find another name.